app/app.vue

```diff
@@ -20,6 +20,7 @@ provide('navigation', navigation)
 </script>
 
 <template>
+  <NuxtLoadingIndicator color="rgb(252,211,77)" />
```

Comment:
We might inherit color from theme. But lets iterate 👍🏼

Reply:
Right that would be best. For some reason couldn't set it using color and with style didn't work either... though didn't try !important